### PR TITLE
Update setup.py

### PR DESCRIPTION
**Pull request by anukaal**
updated classifiers

### CODE REVIEW

**Comment by armenzg** [comment]
@anukaal Hi, what are the implications of those classifiers? Does anybody consume them? Does it set some expectations?

I would say that we run tests against Python 3.6 & some minimal testing of 3.8 (e.g. do packages install at all?).

**Comment by chadwhitacre** [comment]
If 3.6 is the highest version we run our full test suite against, I think we should leave this there for now.

We're trying to get to 3.8 soon though, right?

**Comment by armenzg** [comment]
Should we close then?

> We're trying to get to 3.8 soon though, right?

Correct. Sometime this year.

File: setup.py [hunk 1]
```diff
@@ -165,6 +165,9 @@ def get_requirements(env):
         "Operating System :: POSIX :: Linux",
         "Programming Language :: Python :: 3",
         "Programming Language :: Python :: 3.6",
+        "Programming Language :: Python :: 3.7",
+        "Programming Language :: Python :: 3.8",
+        "Programming Language :: Python :: 3.9",
```
Comment (joshuarli) [inline setup.py:170]: Thanks for the PR, but we can't do this; we're only targetting 3.8 in the future and can't give any guarantees for other versions, though they'll likely Just Work.

When the time comes, I'll change it from 3.6 to 3.8.

**Pull request closed without merging**